docs/developer-guide.md

```diff
@@ -134,18 +134,25 @@ The OS support table in `docs/README.md` must be updates and so does the
 
 ### Bumping ACS version via updatecli
 
-This repo provide experimental support for keeping ACS components versions
-defined in `groups_vars/all.yml` in sync the the ACS releases.
+This repo provide experimental support for updatecli to keep ACS components
+versions in sync with the latest ACS releases. Configurations files are in the
+`scripts/updatecli` folder.
 
-The `updatecli_config.yml` file provides the main pipeline definition and there
+The `updatecli_config.tpl` file provides the main pipeline definition and there
 are multiple values files depending on which acs major version we want to check
 for any new minor release.
 
 Run updatecli with:
 
 ```bash
-updatecli apply --config scripts/updatecli/updatecli_config.yml --values scripts/updatecli/updatecli_acs73.yml
+updatecli apply --config scripts/updatecli/updatecli_config.tpl --values scripts/updatecli/updatecli_acsXX.yml
+updatecli apply --config scripts/updatecli/updatecli_config.tpl --values scripts/updatecli/updatecli_acsXX.yml --values scripts/updatecli/updatecli_tests_amps_target.yml
 ```
```

Comment:
I still need to handle how to skip ags when updating the main group_vars, suggestions welcome

Reply:
you could turn `.target.amps` from a bool to an array of some kind that contains the list of amps present in the `target_file` and then `{{ range }}` on that array in the template to render all the required target (which are all the same expect from the AMP name if I saw correctly)